solana-programs/reward-manager/program/src/processor.rs

```diff
@@ -47,7 +47,8 @@ impl Processor {
         Ok(())
     }
 
-    /// Process example instruction
+    /// Process init instruction
+    /// Initializes the token account and creates a RewardManager account
```

Comment:
might want to specify 'with a minimum number of votes' or smth unless that's clear enough from `state.rs`